docs/swarm/swarm-tutorial/add-nodes.md

```diff
@@ -23,24 +23,24 @@ This tutorial uses the name `worker1`.
 [Create a swarm](create-swarm.md) tutorial step to create a worker node joined to the existing swarm:
 
     ```bash
-    $ docker swarm join --secret 4ao565v9jsuogtq5t8s379ulb \
-      --ca-hash sha256:07ce22bd1a7619f2adc0d63bd110479a170e7c4e69df05b67a1aa2705c88ef09 \
+    $ docker swarm join \
+      --token  SWMTKN-1-49nj1cmql0jkz5s954yi3oex3nedyz0fb0xx14ie39trti4wxv-8vxv8rssmk743ojnwacrr2e7c \
       192.168.99.100:2377
+
+    This node joined a swarm as a worker.
     ```
 
-    If you don't have the command available, you can run the following command:
+    If you don't have the command available, you can run the following command
+    on a manager node to retrieve the join command for a worker:
 
     ```bash
-    docker swarm join --secret <SECRET> <MANAGER-IP>:<PORT>
-    ```
-
-    Replace `<SECRET>` with the secret that was printed by `docker swarm init`
-    in the previous step. Replace `<MANAGER-IP>` with the address of the manager
-    node and `<PORT>` with the port where the manager listens.
+    $ docker swarm join-token worker
 
-    The command generated from `docker swarm init` includes the `--ca-hash` to
-    securely identify the manager node according to its root CA. For the
-    tutorial, it is OK to join without it.
+    To add a worker to this swarm, run the following command:
+        docker swarm join \
+        --token SWMTKN-1-49nj1cmql0jkz5s954yi3oex3nedyz0fb0xx14ie39trti4wxv-8vxv8rssmk743ojnwacrr2e7c \
+        192.168.99.100:2377
```

Comment:
@aaronlehmann can confirm, but this is the current listen address for the manager. This step in the tutorial is not the time to call out the fact that the IP address may be wrong. It's not like they can run the command and substitute "the correct" ip address. They have to handle it when they run `docker swarm init`.

Step 2 of in Create a swarm specifically speaks to this point:
"The --listen-addr flag configures the manager node to listen on port 2377. The other nodes in the swarm must be able to access the manager at the IP address.
"